Frameworks/QuartzCore/CALayer.mm

```diff
@@ -1570,9 +1568,9 @@ - (CATransform3D)sublayerTransform {
 */
 - (void)setBackgroundColor:(CGColorRef)color {
     if (color != nil) {
-        priv->backgroundColor = *[static_cast<UIColor*>(color) _getColors];
+        priv->backgroundColor = *(reinterpret_cast<const __CGColorQuad*>(CGColorGetComponents(color)));
```

Comment:
why not eh

---
In reply to: [113503612](https://github.com/Microsoft/WinObjC/pull/2538#discussion_r113503612) [](ancestors = 113503612)